apps/nextjs/src/components/AppComponents/Chat/chat-lhs-header.tsx

```diff
@@ -17,10 +19,19 @@ const ChatLhsHeader = ({
   isDemoUser,
 }: Readonly<ChatLhsHeaderProps>) => {
   const router = useRouter();
+  const chat = useLessonChat();
 
   return (
     <>
       <div className="mt-6 hidden items-center justify-end gap-5 sm:flex">
+        {process.env.NEXT_PUBLIC_ENVIRONMENT !== "production" && (
+          <div
+            className="flex-grow text-left text-xs"
+            data-testid="chat-aila-streaming-status"
+          >
+            {chat.ailaStreamingStatus}
+          </div>
+        )}
```

Comment:
Are we trying to move away from tailwind ? If so maybe this could be with oak components

Reply:
That's a good point, though in this case I grabbed it from another PR and don't want to cause a conflict there